backend/groth16/groth16.go

```diff
@@ -25,14 +25,14 @@ import (
 	"github.com/consensys/gnark-crypto/ecc"
 	"github.com/consensys/gnark/backend"
 	"github.com/consensys/gnark/backend/witness"
-	"github.com/consensys/gnark/frontend"
-	backend_bls12377 "github.com/consensys/gnark/internal/backend/bls12-377/cs"
-	backend_bls12381 "github.com/consensys/gnark/internal/backend/bls12-381/cs"
-	backend_bls24315 "github.com/consensys/gnark/internal/backend/bls24-315/cs"
-	backend_bls24317 "github.com/consensys/gnark/internal/backend/bls24-317/cs"
-	backend_bn254 "github.com/consensys/gnark/internal/backend/bn254/cs"
-	backend_bw6633 "github.com/consensys/gnark/internal/backend/bw6-633/cs"
-	backend_bw6761 "github.com/consensys/gnark/internal/backend/bw6-761/cs"
+	"github.com/consensys/gnark/constraint"
+	backend_bls12377 "github.com/consensys/gnark/constraint/bls12-377"
```

Comment:
Rename the import alias to `constraint_bls12377` or `cs_bls12377`? And similarly for the rest of the imports.